target/linux/ipq40xx/files/drivers/net/phy/ar40xx.c

```diff
@@ -2042,11 +2057,13 @@ static int ar40xx_probe(struct platform_device *pdev)
 	/* register switch */
 	swdev = &priv->dev;
 
+#ifndef CONFIG_IPQ40XX_PORT_ISOLATION
 	if (priv->mii_bus == NULL) {
 		dev_err(&pdev->dev, "Probe failed - Missing PHYs!\n");
 		ret = -ENODEV;
 		goto err_missing_phy;
 	}
+#endif
```

Comment:
Why is this wrapped? This is a generic error in case no PHY could be found on the bus.

Reply:
I just reverted. Now I did not remove it and tested it on a Frirtz!Box 7530. Everything is fine. :)